### ISSUE TITLE
Suggestion: Allow image uploaded in a DTM properties to be displayed in Cockpit Asset Properties

### ISSUE THREAD

**Issue by rahultsag** [SoftwareAG/cumulocity-digital-twin-manager#32]
![image](https://user-images.githubusercontent.com/116082992/198304591-89ed763d-8128-47eb-9baf-f9f00a68a6ff.png)

**Comment by Vagdevi1998** [comment]
When the image is uploaded for the property through DTM application, user can see the preview of the image in both DTM and cockpit applications. But if users adds the asset properties widget in cockpit and add the associated property, user is able to see the ID of that property instead of image. There is nothing that DTM does as it is a core widget. 

@janhommes Could you please provide your comments on this issue.

**Comment by janhommes** [comment]
yes, this is true. It would need to be implemented into the asset-properties Widget. However, I am not sure if it works with the table layout. User could use a HTML widget atm to display the image.

**Comment by AmarjyotiSAG** [comment]
Hi @rahultsag,
The enhanced Asset Properties 2.0 widget includes support for image previews.
